gui/compsel/comp_explore.m

```diff
@@ -671,12 +671,12 @@ function refresh_comp_explore(h, varargin)
     % erpimage deletes current axis :( - we must use its altered version
     % remember about 'filt'  = [low_boundary high_boundary] option
     if ~h.opt.tri.filt || length(h.spct.freqsel) < 2
-        [~,~,~,~,h.triaxhndls] = erpimage2( h.EEG.icaact(h.comp,:,:) ...
+        [~,~,~,~,h.triaxhndls] = erpimage_silent( h.EEG.icaact(h.comp,:,:) ...
```

Comment:
Erpimage_silent jest chyba tym samym co dwójka bez zmiany axis. Problem
jest taki ze pop prop brzydko działa bez tych domyalnych ustawień.
Idealnie - byłoby gdyby erpimage mógł dostawać strukturę handles i z jej
użyciem plotować gdzie użytkownik chce. To jest jednak tak archaiczny kod,
że mam momentami poczucie, że Twórcy eeglaba znaleźli go w jakiejś
piramidzie. Z tego względu nie chciało mi się grzebać. Na razie mogą być
dwa erpim - a najlepiej gdybyś je połączyła tak ze jakiś argument by
decydował o zachowaniu silent vs 2.
Coś w ten deseń - bo po co dwa razy ten sam kod z innymi tylko kilkoma
linijkami mieć? (nikt od tego nie umrze całe szczęście, także tymczasowo -
no prob).

Ja w tym tyg nie będę dużo wrzucał, najwyzej doszlifuje gałązke z
fastplotem, ale póki co fastplot i eegplot mogą działać równolegle
(docelowo pewnie też - tzn ze mozesz wybrać plotting backed i tak dalej...).
25 mar 2015 11:21 "mzakrzewska" <notifications@github.com> napisał(a):

> In gui/compsel/comp_explore.m
> <https://github.com/mmagnuski/eegDb/pull/70#discussion_r27108296>:
>
> > @@ -671,12 +671,12 @@ function refresh_comp_explore(h, varargin)
> >      % erpimage deletes current axis :( - we must use its altered version
> >      % remember about 'filt'  = [low_boundary high_boundary] option
> >      if ~h.opt.tri.filt || length(h.spct.freqsel) < 2
> > -        [~,~,~,~,h.triaxhndls] = erpimage2( h.EEG.icaact(h.comp,:,:) ...
> > +        [~,~,~,~,h.triaxhndls] = erpimage_silent( h.EEG.icaact(h.comp,:,:) ...
>
> Yes, hopefully więcej poprawek! [image: :smiley:]
> erpimage_silent ma tę zaletę że nie flooduje tekstu?
> erpimage2 tez nie wypluwa nic, a byl tam fix zwiazany z kasowaniem current
> axis - wracam wiec do erpimage2. Comp_exp potrzebuje tych axis
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mmagnuski/eegDb/pull/70/files#r27108296>.
>

-- 

------------------------------